src/components/views/right_panel/UserInfo.tsx

```diff
@@ -165,6 +165,16 @@ function useHasCrossSigningKeys(
     }, [cli, member, canVerify]);
 }
 
+/**
+ * Display one device and the related actions
+ * @param userId current user id
+ * @param device device to display
+ * @param isUserVerified is false when
+ *  - the user is not verified
+ *  - the crypto of `MatrixClient.getCrypto` is unavailable
+ *  - `MatrixClient.getCrypto.getUserVerificationStatus` async call is in progress
```

Comment:
now that `DevicesSection.loading` is true while the async call is in progress, I think this component will no longer be used at all in this scenario?